### PR TITLE
Some small changes

### PR DESCRIPTION
**Pull request by hellwolf**
- Update handler.xkcd.yml
- Update license files
- Updated README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +74,10 @@ If you're interest in peeking under the hood, then check out the contracts packa
   </a>
 </p>
 
+Javascript SDK for building with Superfluid Protocol.
```
Comment (0xdavinchee) [inline README.md]: I think this should be above js-sdk

Comment (hellwolf) [reply README.md]: updated

File: README.md [hunk 2]
```diff
@@ -52,8 +59,20 @@ For technical document, references and tutorials, etc, refer to the
...
 </p>
 
+EVM contracts implementation for the Superfluid Protocol.
+
 If you're interest in peeking under the hood, then check out the contracts package.
 
+#### [`@superfluid-finance/subgraph`](https://github.com/superfluid-finance/protocol-monorepo/tree/dev/packages/subgraph)
+
+<p>
+  <a href="#" target="_blank">
```
Comment (kasparkallas) [inline README.md:69]: Does this do anything?

File: README.md [hunk 3]
```diff
@@ -98,6 +120,17 @@ To build with Superfluid, you can use the Javascript SDK package or the SDK core
 
 SDK-Redux is an application framework for building front-end applications that interact with the Superfluid Protocol.
 
+#### [`@superfluid-finance/hot-fuzz`](https://github.com/superfluid-finance/protocol-monorepo/tree/dev/packages/hot-fuzz)
+
+<p>
+  <a href="#" target="_blank">
```
Comment (kasparkallas) [inline README.md:126]: Same comment as previous

**Review by kasparkallas — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (hellwolf) [reply README.md:69]: not really, leaving as it is for now...

**Review by 0xdavinchee — APPROVED** [review]
LGTM

**Pull request merged**